src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/BurdenEstimates/BurdenEstimateUploadController.kt

```diff
@@ -4,6 +4,7 @@ import org.vaccineimpact.api.app.ResultRedirector
 import org.vaccineimpact.api.app.asResult
 import org.vaccineimpact.api.app.context.ActionContext
 import org.vaccineimpact.api.app.context.RequestDataSource
+import org.vaccineimpact.api.app.errors.BadRequest
```

Comment:
Ah, I didn't realise we had BadRequest as well as InvalidOperationError. They both return 400 status codes. I've been raising an InvalidOperationError when requesting burden estimate csv data from a stochastic set, but it would probably make more sense as a BadRequest. This is an action, so would it make more sense as an InvalidOperation?

Reply:
Good q, not always obvious which to use, but yeah this makes sense :+1: